airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry_entry.py

```diff
@@ -373,16 +422,22 @@ def metadata_entry(context: OpExecutionContext) -> Output[Optional[LatestMetadat
     if not matching_blob:
         raise Exception(f"Could not find blob with etag {etag}")
 
+    metadata_dict = yaml_blob_to_dict(matching_blob)
+    user_identifier = safe_get_slack_user_identifier(airbyte_slack_users, metadata_dict)
+    commit_sha = safe_get_commit_sha(metadata_dict)
+
     metadata_file_path = matching_blob.name
     PublishConnectorLifecycle.log(
         context,
         PublishConnectorLifecycleStage.METADATA_VALIDATION,
         StageStatus.IN_PROGRESS,
         f"Found metadata file with path {metadata_file_path} for etag {etag}",
+        user_identifier=user_identifier,
+        commit_sha=commit_sha,
```

Comment:
The type hints on the `log` method are not accurate, they should be `str | None` instead of `str` for these two parameters.